### PR TITLE
Remove unused methods

### PR DESCRIPTION
**Pull request by katheris**
### Type of change

_Select the type of your PR_

- Refactoring

### Description

* Remove unused methods.
* Move createSecret method to Ca class.

### Checklist

_Please go through this checklist and make sure all applicable tasks have been done_

- [ ] Write tests
- [ ] Make sure all tests pass
- [ ] Update documentation
- [ ] Check RBAC rights for Kubernetes / OpenShift roles
- [ ] Try your changes from Pod inside your Kubernetes and OpenShift cluster, not just locally
- [ ] Reference relevant issue(s) and close them after merging
- [ ] Update CHANGELOG.md
- [ ] Supply screenshots for visual changes, such as Grafana dashboards

### CODE REVIEW

File: operator-common/src/main/java/io/strimzi/operator/common/model/Ca.java [hunk 1]
```diff
@@ -610,13 +611,40 @@ public void createRenewOrReplace(String namespace, String clusterName, Map<Strin
...
+                .withName(name)
+                .withNamespace(namespace)
+                .withLabels(labels)
+                .withAnnotations(annotations)
+                .withOwnerReferences(or)
+                .endMetadata()
+                .withType("Opaque")
+                .withData(data)
+                .build();
+    }
```
Comment (scholzj) [inline operator-common/src/main/java/io/strimzi/operator/common/model/Ca.java:646]: Maybe you can format the indentation to make it more readable?

```suggestion
        return new SecretBuilder()
                .withNewMetadata()
                    .withName(name)
                    .withNamespace(namespace)
                    .withLabels(labels)
                    .withAnnotations(annotations)
                    .withOwnerReferences(or)
                .endMetadata()
                .withType("Opaque")
                .withData(data)
                .build();
    }
```

**Review by scholzj — APPROVED** [review]
One nit. LGTM otherwise.

**Review by tombentley — APPROVED** [review]
Thanks!

**Review by ppatierno — APPROVED** [review]

File: operator-common/src/main/java/io/strimzi/operator/common/model/Ca.java (hunk 1 above)
Comment (katheris) [reply operator-common/src/main/java/io/strimzi/operator/common/model/Ca.java]: Done :)

**Pull request merged**